include/nbl/builtin/hlsl/glsl_compat/subgroup.hlsl

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+#ifndef _NBL_BUILTIN_HLSL_GLSL_COMPAT_SUBGROUP_INCLUDED_
+#define _NBL_BUILTIN_HLSL_GLSL_COMPAT_SUBGROUP_INCLUDED_
+
+#include "nbl/builtin/hlsl/spirv_intrinsics/basic.hlsl"
+
+namespace nbl 
+{
+namespace hlsl
+{
+namespace glsl
+{
+namespace subgroup
```

Comment:
https://github.com/Devsh-Graphics-Programming/Nabla/pull/519/files/1a6e68447b0344e735db02d50fe5cfdf62fcc952..3c19420e94d459d2b35389469a0eaf26a8b65e95#r1293418894

Reply:
should be under `subgroup_basic` or `subgroup/basic`